Assets/Scripts/Framework/Gameplay/Interactions/Grabbables/Grabbable.cs

```diff
@@ -30,9 +33,33 @@ private protected override bool Interact(Interactor interactor)
                 ? interactor
                 : null;
 
-            interactor.SetInteractingTarget(this, IsBeingInteracted);
+            UseGrabbable(interactor);
 
             return true;
         }
+
+        [Server]
```

Comment:
If this `[Server]` attribute works as we expect, we can add it to the method `Interact` in the base class `Interactor` and in this class.